Directory.Build.props

```diff
@@ -25,7 +25,12 @@
 
   <!-- The TFMs to build and test against. -->
   <PropertyGroup>
+    <NetFrameworkMinimum>net462</NetFrameworkMinimum>
+    <NetCurrent>net9.0</NetCurrent>
```

Comment:
This overwrites Arcade. I think Arcade would be using net10.0 now.

Reply:
OK removing NetCurrent.